pkg/ingester/tailer.go

```diff
@@ -17,7 +17,10 @@ import (
 	util_log "github.com/grafana/loki/pkg/util/log"
 )
 
-const bufferSizeForTailResponse = 5
+const (
+	bufferSizeForTailResponse = 5
+	maxDroppedStreams         = 10 // make tailer.droppedStreams slice bounded.
```

Comment:
I'd suggest removing this comment here as it's not serving much purpose and will go stale

Reply:
Should we make this configurable?

Reply:
> Should we make this configurable?
Hmm. good point. But not sure though.

Also where do we put the config?. Two options I see, 

1. In ingester itself (where this tailer component is part of) - `ingester.tailer.max-dropped-streams` (or something similar)
2. In LogCLI as a part of `--tail` web socket request. - `logcli --tail --max-dropped-streams=200`

both has own pros and cons. (2) sounds reasonable, but then `client` can make ingester OOM by setting too high value. 

I would prefer option (1).

Reply:
Option 1, for sure. We shouldn't make the client responsible for protecting the server from overload.